src/containers/ConnectorManagement.jsx

```diff
@@ -105,23 +107,15 @@ export default class ConnectorManagement extends Component {
   }
 
   gotoParent () {
-    this.clearLoginFormInputs()
-
-    const router = this.context.router
-    let url = router.location.pathname
-    router.push(url.substring(0, url.lastIndexOf('/')))
-  }
-
-  /*
-   * Empies Login Form values to prevent the browser (aka firefox) to propose to save the credentials
-   */
-  clearLoginFormInputs() {
-    const formElements = document.querySelectorAll(`.${stylesLoginForm['account-form-login']} input`)
-    formElements.forEach(input => {
-      if (input.type === "text" || input.type === "password") {
-        input.value = ""
-      }
-    })
+    this.setState({isClosing: true})
+
+    // The setTimeout allows React to perform setState related actions
```

Comment:
If I understand well, without this `setTimeout`, React does not have time to update virtual DOM with new empty values for `login` and `password` ?